src/vs/platform/update/electron-main/updateMetadataProvider.ts

```diff
@@ -0,0 +1,59 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// {{SQL CARBON EDIT}} - This file contains interfaces, util functions and url definitions
+// to support Download Center based update.
+import { IProductService } from 'vs/platform/product/common/productService';
+import { IUpdate } from 'vs/platform/update/common/update';
+
+export const updateMetadataUrl = 'https://go.microsoft.com/fwlink/?linkid=2274438';
```

Comment:
Can you add a comment explaining why there are different URLs for each Mac target, but Windows and Linux are combined?

Reply:
added comment in the field definition in `product.ts`